src/reconcile/rec_write.c

```diff
@@ -6591,3 +6594,37 @@ __rec_dictionary_lookup(
 	*dpp = next;
 	return (0);
 }
+
+/*
+ * __verbose_lookaside_write --
+ *	Create a verbose message to display with details about the transaction
+ * state when performing a lookaside table write.
+ */
+static void
+__verbose_lookaside_write(WT_SESSION_IMPL *session, uint64_t inserted_cnt)
+{
+#ifdef HAVE_VERBOSE
+	WT_SESSION *wt_session;
+	uint64_t pinned_range;
+	uint32_t pct_dirty, pct_full;
+
+	if (WT_VERBOSE_ISSET(session, WT_VERB_LOOKASIDE)) {
+
+		wt_session = &session->iface;
+
+		(void)__wt_eviction_clean_needed(session, &pct_full);
+		(void)__wt_eviction_dirty_needed(session, &pct_dirty);
+		wt_session->transaction_pinned_range(wt_session, &pinned_range);
+
+		__wt_verbose(session, WT_VERB_LOOKASIDE,
+		    "Page reconciliation required %" PRIu64
+		    " lookaside entries: %" PRIu64
+		    " pinned TXN ID's, cache is %" PRIu32
+		    "/%" PRIu32 "%% dirty/clean",
+		    inserted_cnt, pinned_range, pct_dirty, pct_full);
```

Comment:
The formatting looks inconsistent to me here, I think it'll come out as:

```
Page reconciliation required 99 lookaside entries: 99 pinned TXN ID's, cache is 99/99% dirty/clean
```

It'd be better as:

```
Page reconciliation triggered lookaside write. Entries written: 99, pinned transaction IDs: 99, cache dirty: 99%, cache clean: 99%
```

That uses common separators between terms. There are also other grammar corrections.